stopify-continuations/src/callcc/jumper.ts

```diff
@@ -325,6 +330,10 @@ const jumper = {
 
         path.node.body.body.push(ifConstructor);
       }
+
+      // Increment the remainingStack at the last line of the function.
+      // This does not break tail calls.
+      path.node.body.body.push(increaseStackSize);
```

Comment:
I don't understand. You're adding `stackSize++` to the end of the function. What if the end of the function is never reached?

Reply:
Then it’s dead code. This to handle functions that don’t have an explicit return.